clifford/_layout.py

```diff
@@ -690,6 +690,29 @@ def leftLaInvJIT(value):
 
         return leftLaInvJIT
 
+    @_cached_property
+    def pinv_func(self):
+        """
+        Get a function that returns left-pseudoinverse using a computational
```

Comment:
Is the left pseudo-inverse equal to the right pseudo-inverse (i.e., can you find a counterexample where it isn't?)

Reply:
Good question. I can't come up with a counter example. Something interested and perhaps related I noticed is that if I replace the original np.linalg.solve with np.linalg.lstsq I get for he test example
```
Expected :0.5 + (3002399751580329.5^e2) - (0.5^e3) - (0.75^e13) - (3002399751580329.5^e23)
Actual   :0.5 - (0.5^e3)
```
I think it is save to say that this is undesired behavior. The pinv function behaves much gentler. And in the numpy docs I see no mention of left and right actually, so perhaps we should leave that distinction altogether? Since it seems to return the input element anyway in case of singular behavior I don't see how that could be different left and right.